File: iCovieAdmin-Ng/src/app/core/auth/_models/Masters/neighbourhoodcategory.model.ts


export class NeighbourhoodCategoryModel {
    id: number;    
    neighbourhoodcategory:string;
    shortname:string;   
    icon:string;   
    cid:number; 
    isCore: boolean = false;    
    clear(): void {
        this.id= undefined;
        this.neighbourhoodcategory= '';        
        this.cid = undefined;      
        this.isCore=false;
	}
}

File: iCovieApi/iCovieApi/Models/Master/NeighbourhoodCategoryModel.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace iCovieApi.Models
{
    public class NeighbourhoodCategoryModel
    {
        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int id { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string neighbourhoodcategory { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string icon { get; set; }

        


        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int cid { get; set; }
    }
}